ai-ml/trainium-inferentia/addons.tf

```diff
@@ -247,21 +250,21 @@ resource "kubectl_manifest" "torchx_etcd" {
   depends_on = [module.eks.eks_cluster_id]
 }
 
-#---------------------------------------------------------------
-# Volcano Schduler for TorchX
-# NOTE: This will be replaced with Helm Chart deployment with eks_data_addons
-#---------------------------------------------------------------
-data "http" "volcano_development_yaml" {
-  url = "https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml"
-}
+# #---------------------------------------------------------------
+# # Volcano Schduler for TorchX
+# # NOTE: This will be replaced with Helm Chart deployment with eks_data_addons
+# #---------------------------------------------------------------
+# data "http" "volcano_development_yaml" {
+#   url = "https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml"
+# }
 
-resource "kubectl_manifest" "volcano" {
-  yaml_body = <<-YAML
-    ${data.http.volcano_development_yaml.response_body}
-  YAML
+# resource "kubectl_manifest" "volcano" {
+#   yaml_body = <<-YAML
+#     ${data.http.volcano_development_yaml.response_body}
+#   YAML
 
-  depends_on = [module.eks.eks_cluster_id]
-}
+#   depends_on = [module.eks.eks_cluster_id]
+# }
```

Comment:
remove commented lines

Reply:
done as well, i am sorry I had to do git add . to remove all the wrong subfolders but wont happen again should be ok now